tests/samples/pad_pipeline_vectorized.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: iree-compile --iree-hal-target-backends=amd-aie --compile-to=executable-sources %s | iree-opt --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-hal-translate-target-executable-variants{target=amd-aie})))" --iree-amdaie-use-pipeline=pad | FileCheck %s
```

Comment:
You dont need to start all the way from `iree-compile` . You should be able to write a lit test just for this pass and this does not go into `tests/samples`. That pass test should go into https://github.com/nod-ai/iree-amd-aie/tree/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test .